WebJobs.Script.sln

```diff
@@ -448,6 +448,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "HttpTrigger-TypeScript", "H
 		sample\HttpTrigger-TypeScript\index.ts = sample\HttpTrigger-TypeScript\index.ts
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WebJobs.Script.Scaling", "src\WebJobs.Script.Scaling\WebJobs.Script.Scaling.csproj", "{292A9038-5807-4095-B37F-9A77170D7402}"
```

Comment:
Ultimately I should probably move some of the other scale stuff that I have already (e.g. HostPerformanceManager) into this assembly.